### PR TITLE
[chore](hint) update group commit plan fail hint

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/planner/CloudGroupCommitPlanner.java [hunk 1]
```diff
@@ -74,7 +74,11 @@ protected void selectBackends(ConnectContext ctx) throws DdlException {
             }
         }
 
-        throw new DdlException("No suitable backend for cloud cluster=" + cluster);
+        List<String> backendsInfo = backends.stream()
+                .map(be -> "{ beId=" + be.getId() + "alive=" + be.isAlive() + ", active=" + be.isActive()
+                        + ", decommission=" + be.isDecommissioned() + " }")
+                .collect(Collectors.toList());
+        throw new DdlException("No suitable backend for cloud cluster=" + cluster + ", backends = " + backendsInfo);
```
Comment (dataroaring) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/planner/CloudGroupCommitPlanner.java:81]: limit numer 4 at most?

Comment (yujun777) [reply fe/fe-core/src/main/java/org/apache/doris/cloud/planner/CloudGroupCommitPlanner.java:81]: 单个cluster 且所有be都失败情况下，不会存在很多be吧

**Comment by yujun777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by yujun777** [comment]
run feut

**Pull request merged**